swig/include/python/docs/osr_spatialreference_docs.i

```diff
@@ -1,5 +1,26 @@
 %feature("docstring") OSRSpatialReferenceShadow "
 Python proxy of an :cpp:class:`OGRSpatialReference`.
+
+Create a new spatial reference object. An empty object will be created
+unless exactly one of the following parameters is provided.
+
+Parameters
+----------
+epsg : int, optional
+    EPSG CRS code, as understood by :py:meth:`ImportFromEPSG`
+esri : list, optional
+    ESRI ``.prj`` text, as understood by :py:meth:`ImportFromESRI`
+proj4 : str, optional
+    PROJ CRS string, as understood by :py:meth:`ImportFromProj4`
+wkt : str, optional
+    WKT CRS string, as understood by :py:meth:`ImportFromWkt`
```

Comment:
can we add PROJJSON to it, both serialized or as a dict ?

Reply:
> What is the import function for that?

SetFromUserInput(). Cf test_osr_import_projjson in osr_basic.py
I think the simplest would be to change OSRNewSpatialReference() / OGRSpatialReference(const char*) constructor to delegate to SetFromUserInput() which would include the current importFromWkt() behavior